OpenRA.Mods.RA/Traits/GpsRemoveFrozenActor.cs

```diff
@@ -35,21 +34,31 @@ public GpsRemoveFrozenActor(Actor self, GpsRemoveFrozenActorInfo info)
 			watchers = self.World.ActorsWithTrait<GpsWatcher>().Select(w => w.Trait).ToArray();
 		}
 
-		public bool RemoveActor(Actor self, Player owner)
+		public void Tick(Actor self)
```

Comment:
Nope I'm being dumb here. I'll probably need to wire up a way for the GPS to signal when the GPS grant status changes so this trait can re-evaluate stuff.